src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -975,19 +976,22 @@ void WriteLog<I>::aio_read_data_blocks(
     [this, log_entries, bls, ctx](int r) {
       for (unsigned int i = 0; i < log_entries.size(); i++) {
         bufferlist valid_data_bl;
-        auto length = log_entries[i]->is_write() ? log_entries[i]->write_bytes :
-                                                   log_entries[i]->ws_datalen;
+        auto entry = static_pointer_cast<WriteLogEntry>(log_entries[i]);
```

Comment:
Elsewhere in `ssd/WriteLog.cc` it is named `write_entry`.  The types are are confusing here so let's keep the names consistent.
```suggestion
        auto write_entry = static_pointer_cast<WriteLogEntry>(log_entries[i]);
```

Reply:
done